src/language/tekster_en.ts

```diff
@@ -511,8 +514,10 @@ export default {
   'utdanning.spm.mål': 'What is your goal for taking this education?',
   'utdanning.alert-tittel.mål':
     'You must provide documentation of the education you are taking or are going to take',
-  'utdanning.alert-beskrivelse.mål':
-    'The documentation must indicate:<ul><li>the name of the place of study</li><li>the name of the course or education</li><li>how much you are going to study</li><li>the period in which you are going to study</li></ul><br/><br/>The documentation must show clearly to whom it pertains. If you are already receiving transitional benefit and are applying for an extension of the benefit period because you have been offered a place on a course of education, the documentation must also show the date on which you accepted the offer.',
+  'utdanning.alert-beskrivelse.mål.skolepenger':
+    'The documentation must indicate:<ul><li>the name of the place of study</li><li>the name of the course or education</li><li>how much you are going to study</li><li>the period in which you are going to study</li></ul><br/>The documentation must show clearly to whom it pertains.',
```

Comment:
Kanskje ikke like presist, men lettere å forstå?
```suggestion
    'The documentation must indicate:<ul><li>the name of the place of study</li><li>the name of the course or education</li><li>how much you are going to study</li><li>the period in which you are going to study</li></ul><br/>The documentation must clearly show who it concerns.',
```

Reply:
Er det ikke litt forskjell på - inneholder og angår her? 
Concerns - da tenker jeg på hvem som skal lese dokumentasjonen, ikke hvem dokumentasjonen "dokumenterer". 
Pertains er jo et presist ord selv om det er litt vanskeligere?

Men jeg er ikke ekspert på engelsk. Ikke så bra norsk heller :D ☝️